app/views/characters/_character_weapons.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row characters-row">
+	<% @character_weapons.each do |character_weapon| %>
+		<div class="col-sm-4 characters-col">
+			<div class="card">
+			  <div class="card-block">
+			    <h4 class="card-title"><%= character_weapon.weapon.name %></h4>
+			    <ul class="card-text">
+					<li>Damage: <%= character_weapon.damage %></li>
+					<li>Crit: <%= character_weapon.crit %></li>
+			    </ul>
+			    <%= link_to "Show", character_weapon_path(character_weapon), class: "btn access_button" %>
+			    <%= link_to "Edit", edit_character_weapon_path(character_weapon), class: "btn btn-primary" %>
+			    <%= link_to 'Destroy', character_weapon, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger" %>
+			  </div>
+			</div>
+		</div>
+	<% end %>
+</div>
+<%= link_to 'New Weapon', new_character_weapon_path(character_id: @character.id), class: 'btn btn-primary' %>
```

Comment:
I see, since `CharacterWeapon` is a _children_ of `Character` resource, we need to change `CharacterWeaponsController` to be a nested resource:

In your routes.rb
```ruby
resources :characters do
   resources :character_weapons
end
```

That way all character weapons paths will be routed to a character resources. You can check the new routes with:

```bash
rake routes
```